scripts/install-helpers/install.sh

```diff
@@ -5,7 +5,12 @@ OCP_PULL_SECRET_LOCATION="${OCP_PULL_SECRET_LOCATION:-$HOME/pull-secret.json}"
 MIRRORING=false
 ADD_IMAGE_PULL_SECRET=false
 GA_RELEASE=true
-TRUSTEE_IMAGE=${TRUSTEE_IMAGE:-quay.io/openshift_sandboxed_containers/kbs:v0.10.1}
+ITA_KEY="${ITA_KEY:-}"
+DEFAULT_IMAGE=quay.io/openshift_sandboxed_containers/kbs:v0.10.1
+if [ -n "$ITA_KEY" ]; then
+    DEFAULT_IMAGE+="-ita"
+fi
+TRUSTEE_IMAGE=${TRUSTEE_IMAGE:-$DEFAULT_IMAGE}
```

Comment:
nit: May be we can use the following two liner?
```
DEFAULT_IMAGE="quay.io/openshift_sandboxed_containers/kbs:v0.10.1${ITA_KEY:+-ita}"
TRUSTEE_IMAGE="${TRUSTEE_IMAGE:-$DEFAULT_IMAGE}"
```

Reply:
> DEFAULT_IMAGE="quay.io/openshift_sandboxed_containers/kbs:v0.10.1${ITA_KEY:+-ita}"

IMHO, this is harder to read than the expanded version, but I can use it if you prefer.

Nowadays, with the amount of different things I'm having to deal with, the more explicit a code is, the better. :-)

Reply:
Ha ha.. let it be then :-) .. I don't have any hard preference..